physical_properties/pKa/experimental_data/experimental_ID_and_SAMPL6_molecule_ID_table.csv

```diff
@@ -0,0 +1 @@
+Experimental Molecule ID,SAMPL6 Molecule ID
M01,SM01
M02,SM02
M03,SM03
M04,SM04
M05,SM05
M06,SM06
M07,SM07
M08,SM08
M09,SM09
M10,SM10
M11,SM11
M12,SM12
M13,SM13
M15,SM14
M16,SM15
M18,SM16
M19,SM17
D01,SM18
D02,SM19
D05,SM20
D06,SM21
D07,SM22
D08,SM23
D09,SM24
```

Comment:
Why is this only one line? I would guess if people are going to look at this it needs to be human readable? Could you separate these by molecule or at least add a comment at the top explaining how this file is organized?

Reply:
It is supposed to be a CSV file of two columns. I fixed this problem.